src/main/java/io/odpf/depot/bigquery/client/BigQueryClient.java

```diff
@@ -33,7 +34,7 @@ public class BigQueryClient {
     @Getter
     private final TableId tableID;
     private final BigQuerySinkConfig bqConfig;
-    private final BQTableDefinition bqTableDefinition;
```

Comment:
we should keep this class and add a clustering config there. 
no need to complicate and move static config code to different classes.